index.html

```diff
@@ -270,87 +329,289 @@
+variety of data types, and at intensifying speed is leading to an unprecedented concentration of power
+that threatens private and public liberties.
+
+What's more, automation and the increasing computerisation of all aspects of our lives
+both increase the power of information and decrease the cost of a number of intrusive
+behaviours that would be more easily kept in check if the perpetrator had to be in the same room as
+the victim.
+
+These asymmetries of information and of automation create commanding asymmetries of power. It is not
+so much that <q>code is law</q>  but rather that code is judge, jury, and executioner.
+
+The Web is for everyone ([[?For-Everyone]]). It is <q>a platform that helps people and provides a
+net positive social benefit</q> ([[?ETHICAL-WEB]], [[?design-principles]]). One of the ways in which the
+Web serves people is by protecting them in the face of asymmetries of power and this includes
+establishing and enforcing rules to govern the power of data.
+
+The Web is a social and technical system made up of [=information flows=]. Because this document
+is specifically about [=privacy=] as it applies to the Web, it also focuses on privacy around
+information flows.
+
+<dfn data-lt="governance">Data governance</dfn> is the system of [=rules=] that regulate [=information flows=]. When
+[=people=] are involved in these [=information flows=], [=data governance=] determines how
+these rules constrain and distribute the power of information between different [=actors=].
+The <dfn>rules</dfn> describe the way in which different [=actors=] may, must,
+or must not produce or [=process=] flows of information from, to, or about other [=actors=]
+([[?GKC-Privacy]], [[?IAD]]).
+
+<dfn>Privacy</dfn> is achieved in a given [=context=] that either involves [=personal data=] or
+involves information being presented to [=people=], and the [=rules=] of that [=context=] are
+followed appropriately. When the [=rules=] for that [=context=] are not followed, there is a
+<dfn>privacy violation</dfn>. Similarly, we say that a particular interaction is
+<dfn data-lt="appropriately">appropriate</dfn> when the [=rules=] are adhered
+to) or <dfn data-lt="inappropriately">inappropriate</dfn> otherwise.
+
+Typically, <dfn>actors</dfn> are [=people=] but they can also be collective endeavours such as
+companies, associations, or political parties (eg. in the case of espionage), which are known as
+[=parties=]. It is important to keep in mind that not all people are equal in how they may resist
+the imposition of unfair [=rules=], some [=people=] are more [=vulnerable=] and therefore in greater
+need of protection. The focus of this document is on the impact that leveraging this power
+differential can have against people, but the same techniques are also used to leverage the power of
+information against other entities, such as companies or governments.
```

Comment:
Good point. Now: "The focus of this document is on the impact that this power differential can
have against people, but it can also impact other entities, such as companies or governments."